cpp/CMakeLists.txt

```diff
@@ -104,6 +104,14 @@ if (${USE_BOOST} STREQUAL "ON")
   include_directories (${Boost_INCLUDE_DIRS})
 endif ()
 
+if (${NOT_USE_STDMUTEX} STREQUAL "ON")
```

Comment:
Can you add these to the options above (line 80 and following)?

Reply:
done